server/portal/apps/datafiles/views.py

```diff
@@ -38,6 +38,12 @@ def get(self, request):
         default_system = user_systems[settings.PORTAL_DATA_DEPOT_LOCAL_STORAGE_SYSTEM_DEFAULT]
         response['default_host'] = default_system['host']
 
+        for system in response['system_list']:
```

Comment:
do you need to check for only `tapis` systems?

Reply:
I think this does effectively limit this to tapis systems.

Reply:
Okay now accounts for non tapis api systems such as googledrive